fixtures/chauvet-dj/hurricane-haze-2d.json

```diff
@@ -0,0 +1,109 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Hurricane Haze 2D",
+  "categories": ["Hazer"],
+  "meta": {
+    "authors": ["Tryll"],
+    "createDate": "2022-11-14",
+    "lastModifyDate": "2022-11-14"
+  },
+  "links": {
+    "manual": [
+      "https://www.chauvetdj.com/wp-content/uploads/2015/12/Hurricane_Haze_2D_UM_Rev12_WO-1.pdf"
+    ],
+    "productPage": [
+      "https://www.chauvetdj.com/products/hurricane-haze-2d/"
+    ]
+  },
+  "physical": {
+    "dimensions": [267, 350, 285],
+    "weight": 8.5,
+    "power": 533,
+    "DMXconnector": "3-pin"
+  },
+  "availableChannels": {
+    "Blower Speed": {
+      "capabilities": [
+        {
+          "dmxRange": [0, 5],
+          "type": "Generic",
+          "comment": "OFF"
+        },
+        {
+          "dmxRange": [6, 75],
+          "type": "Generic",
+          "comment": "Level 1"
+        },
+        {
+          "dmxRange": [76, 125],
+          "type": "Generic",
+          "comment": "Level 2"
+        },
+        {
+          "dmxRange": [126, 165],
+          "type": "Generic",
+          "comment": "Level 3"
+        },
+        {
+          "dmxRange": [166, 185],
+          "type": "Generic",
+          "comment": "Level 4"
+        },
+        {
+          "dmxRange": [186, 255],
+          "type": "Generic",
+          "comment": "Level 5"
+        }
+      ]
+    },
+    "Haze Volume": {
```

Comment:
`Fog` is the correct capability type; `FogOutput` is intended for capabilities that set the fog output volume without actually activating the fog.

I think we can use `20%`/`40%`/`60%`/`80%`/`100%` instead of levels 1/2/3/4/5 for both channels.